code/modules/vehicles/arc/arc.dm

```diff
@@ -17,11 +17,10 @@
 
 	interior_map = /datum/map_template/interior/arc
 
-	passengers_slots = 3
-	xenos_slots = 5
+	passengers_slots = 2 // 5 total. Reserved slots are added to passenger slots.
+	xenos_slots = 4
 
 	entrances = list(
-		"left" = list(2, 0),
 		"right" = list(-2, 0),
```

Comment:
Is it possible to make two tiles valid spaces to enter the vehicle? Visually this turf should be a good spot to enter the vehicle to me: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/6c1a26ea-8b00-43b8-abac-c3cd7b0e4a82)

Reply:
Doesn't seem to be doable without reworking a fair bit of entrance code unfortunately